src/app/views/sidebar/sample-queries/SampleQueries.tsx

```diff
@@ -300,6 +308,16 @@ export class SampleQueries extends Component<ISampleQueriesProps, any> {
   }
 
 }
+function displayTipMessage(actions: any, selectedQuery: any) {
```

Comment:
I think we have a Query type that you can use here instead of setting it to any.